src/status_im/protocol/listeners.cljs

```diff
@@ -9,7 +9,8 @@
 (defn- parse-payload [payload]
   (debug :parse-payload)
   (try
-    {:payload (r/read-string (u/to-utf8 payload))}
+    (let [read (comp r/read-string u/to-utf8 u/to-utf8)]
+      {:payload (read payload)})
```

Comment:
if i add 
```clojure
(debug payload "\n\n"
            (u/to-utf8 payload) "\n\n"
            (u/to-utf8 (u/to-utf8 payload)) "\n\n")
```
here https://github.com/status-im/status-react/blob/5c7fd28b29d2513712cb25de00e603c04736a035/src/status_im/protocol/listeners.cljs#L11-L11 result is
```
0x307837623361366436353733373336313637363532643639363432303232333133343339333133343338333433323332333833383332333532643330363233373335333333383633333332643635363633343334326433353333333936323264363136323334363232643635333533343338333936313335333433313632363333333232326332303361373236353731373536393732363537333264363136333662336632303734373237353635326332303361373437393730363532303361366436353733373336313637363532633230336137343639366436353733373436313664373032303331333433393331333433383334333233323338333833323335326332303361363336663665373436353665373432303232333733373337323232633230336136333666366537343635366537343264373437393730363532303232373436353738373432663730366336313639366532323263323033613633366336663633366232643736363136633735363532303332326332303361373336383666373733663230373437323735363537643061

0x7b3a6d6573736167652d69642022313439313438343232383832352d30623735333863332d656634342d353339622d616234622d653534383961353431626333222c203a72657175697265732d61636b3f20747275652c203a74797065203a6d6573736167652c203a74696d657374616d7020313439313438343232383832352c203a636f6e74656e742022373737222c203a636f6e74656e742d747970652022746578742f706c61696e222c203a636c6f636b2d76616c756520322c203a73686f773f20747275657d0a

{:message-id "1491484228825-0b7538c3-ef44-539b-ab4b-e5489a541bc3", :requires-ack? true, :type :message, :timestamp 1491484228825, :content "777", :content-type "text/plain", :clock-value 2, :show? true}
```

I don't know what exactly has been changed in `web3` or `go-ethereum`, but now we have to call `u/to-utf8` twice. I will add a comment as a reminder that we need to revisit this in future.